src/event.rs

```diff
@@ -370,6 +370,11 @@ pub enum WindowEvent<'a> {
     ///
     /// At the moment this is only supported on Windows.
     ThemeChanged(Theme),
+
+    /// An entry in the active menu has been activated.
+    ///
+    /// The event contains the `id` of the entry (given by the user when it was created).
+    MenuEntryActivated(isize),
```

Comment:
It could be interesting to add a new `SystemEvent` variant instead of using `WindowEvent` (we could include `UrlReceived`).